### PR TITLE
Add the project

### PR DESCRIPTION
**Pull request by farouqdaif1**
## Review by Hamza

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,60 +1,58 @@
...
+import addItemUI, { diplayTask, clear } from './uImanpulate.js';
+import Task from './task.js';
+import getDataLS, { addTaskLS, indexValue, remove } from './localstorage.js';
+import editText from './add-remove.js';
 
-const tasks = [
-  { description: 'wash the dashes', completed: false, index: 0 },
-  { description: 'complete to do ', completed: false, index: 1 },
-  { description: 'wash the dashes', completed: false, index: 2 },
-];
```
Comment (Lockless404) [inline src/index.js:9]: The data should be take from the user and not hard coded.

File: src/index.js [hunk 2]
```diff
@@ -1,60 +1,58 @@
...
+});
+list.addEventListener('change', (e) => {
+  if (e.target.classList.contains('check')) {
+    const task = document.querySelector(`#${e.target.id}`);
+    const taskx = e.target;
+    statueUdpdateLS(taskx);
+    statueUdpdateUI(task);
+  }
+});
+list.addEventListener('click', (e) => {
```
Comment (Lockless404) [inline src/index.js:28]: Only one event listener should be added to list. You can use if statements to declare the target.

File: src/localstorage.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+// get the data from local storage
+export default function getDataLS() {
```
Comment (Lockless404) [inline src/localstorage.js:2]: Use classes to streamline your functions.

**Review by Lockless404 — CHANGES_REQUESTED** [review]
Good job so far! 
There are some issues that you still need to work on. 

Required Changes ♻️
Check the comments under the review.

File: src/index.js (hunk 1 above)
Comment (farouqdaif1) [reply src/index.js:9]: It is already deleted code

File: src/index.js (hunk 2 above)
Comment (farouqdaif1) [reply src/index.js:28]: ok but I want it to hear two actions not the same action

File: src/index.js (hunk 1 above)
Comment (farouqdaif1) [reply src/index.js:9]: Thank you

File: src/localstorage.js (hunk 3 above)
Comment (farouqdaif1) [reply src/localstorage.js:2]: Thank you

**Pull request merged**